### ISSUE TITLE
1.7 Strange interaction between Ex Nihilo and Expanded Redstone.

### ISSUE THREAD

**Issue by artaficialAI** [reteo/Survival-Industry#143]
If the player is holding an Ex Nihilo hammer, when an Expanded Redstone Block Breaker breaks a block, the block breaker will output the block's drop as if the block had been broken by an Ex Nihilo hammer.

[Video of the interaction](https://youtu.be/ozt1f7u6GSo)

**Comment by artaficialAI** [comment]
I'm not sure if this is intended or not, but it does make the early game of the skyblock much less tedious.  As it stands right now the skyblock early game is really slow.

**Comment by reteo** [comment]
Of course, it probably is something Reika will want to know about.  I've submitted the issue to him for triage in ReikaKalseki/Reika_Mods_Issues#1794

**Comment by reteo** [comment]
This is probably not something that can be fixed without rewriting Ex Nihilo, so we'll just call it a feature. ;)
